frontend/public/components/pod.tsx

```diff
@@ -695,6 +695,7 @@ export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
                       pod.metadata.name,
                       pod.metadata.namespace,
                     )}/containers/${container.name}/debug`}
+                    data-test-id={`popup-debug-container-link-${index}`}
```

Comment:
Prefer `data-test` for new test IDs. It would be better to use the container name than the index here.